Dream Catchers/Assets/_Game/Scripts/AI/BallonPatrol_State.cs

```diff
@@ -0,0 +1,59 @@
+﻿using UnityEngine;
+using System.Collections;
+
```

Comment:
RequireComponent[typeof(FSM)]

- Not required but that line will ensure that when this script gets attached to an object it will also attach the fsm script.